src/en/space-station-14/departments/medical/proposals/Virology Reworked.md

```diff
@@ -0,0 +1,72 @@
+Alternatively, virologists can attempt to create vaccines to the virus in question, in order to create a vaccine, the virologist would need to provide multiple samples of the virus to the vaccinatior machine, which will then synthesize a vaccine with a hidden success probability based on the volatility of the target virus. Vaccine production will require *something* to actually create the vaccine. 
+A successful vaccine will prevent the specific virus the vaccine is created for from jumping to any person who has not already been infected by the vaccine parent, but will not prevent any future mutation of the same virus eventually coming around and infecting later.
+
+## External Infection Sources
+Viruses that don't originate in the crew round start can be instead picked up from other sources, such as raw meats in the kitchen, not cleaning meat cutting tools in the kitchen, rats and mice, artifacts creating (usually very lethal) diseases, and sometimes external sources, such as carp bites, vent spider bites, spawned blood puddles, and potentially bio terror from nukkies.
+This is mostly done so the crew can't somehow systematically wipe out viruses.
+
+## Viral Modification
+Virologists can take multiple viruses and attempt to take characteristics from each of them in an effort to create new viruses, which can potentially be used to combat other viruses, or be turned into bioweappns.
+Viruses can be modified when exposed to Mutagen or Spacacillin to cause different effects.
+Mutagen will immediately trigger a mutation of the virus as would normally happen when the virus transfers between subjects. 
+Spacacillin can be used to "stabilize" the virus, and then later be used to intentionally infect people without the virus that the original sample was taken from. A Spacacillin modified virus would block the original virus from transferring to the patient as an alternative to a vaccine, as a form of inoculation instead but still subject the patient to the symptoms of the virus.
+
+## Contagion Tiers
+These can all stack in one virus based on volatility and mutation, and will always start with at least one of these.
+
+Blood to Blood: Requires direct blood contact to transfer, such as repeated syringe use, or the same melee weapon used against multiple people. Drinking blood may or may not count. Very low point cost.
+Saliva: Would require someone to drink or ingest the virus from an item someone else has drank or eaten off of without being cleaned first. Low point cost.
+Airborne, Inhalation: Would require people to stand near each other in order to transfer the virus. Can be blocked with breath masks. Moderate point cost.
+Contact: Virus can be picked up through contact with objects the infected has handled with a lifespan assigned to the virus for objects, as well as hugs, cuffing actions, buckling. Moderate point cost.
```

Comment:
I imagine some kinds of clothing could partially or completely counteract the contact-based contagion, for example latex/nitrile gloves could do that (so they have a use).

Reply:
Yeah, wasn't entirely sure wether to include those on first pass.